tests/test_arraycreation.py

```diff
@@ -516,6 +517,8 @@ def test_vander_seq(sequence):
     assert_allclose(vander_func(numpy, sequence), vander_func(dpnp, sequence))
 
 
+# Expected warning
+@pytest.mark.filterwarnings("ignore::numpy.ComplexWarning")
```

Comment:
We have the existing f/w to suppress that:
```suggestion
@pytest.mark.usefixtures("suppress_complex_warning")
```

Reply:
I overlooked it. Thanks
The warnings on Gen9 will be handled too